.fixtures.yml

```diff
@@ -36,5 +36,8 @@ fixtures:
     archive:
       repo: git://github.com/voxpupuli/puppet-archive.git
       ref: 'v3.0.0'
+    windows_env:
```

Comment:
Why is this needed?

Reply:
This is needed to set `PATH` for sensuctl on Windows since the install isn't a package, it's just a zip file.